opm/parser/eclipse/Parser/Parser.hpp

```diff
@@ -101,6 +102,29 @@ namespace Opm {
             addParserKeyword( std::unique_ptr< ParserKeyword >( new T ) );
         }
 
+        // below are some helper functions
+
+        static EclipseState parse(std::shared_ptr<const Deck> deck, const ParseContext& context = ParseContext());
```

Comment:
It doesn't have to exist, but I added it primarily to have a single line of attack;  If you want to create an EclipseState, no matter if you have deck(ptr), context, file or string, you would resort to `Parser::parse`.

I'm open for removing it.